android/src/main/java/com/reactnativebledidcomm/peripheral/PeripheralManager.kt

```diff
@@ -63,8 +63,12 @@ class PeripheralManager(
     }
 
     fun shutdownPeripheral() {
-        this.gattServer.connectedDevices.clear()
+        val connectedDevices = this.bluetoothManager.getConnectedDevices(BluetoothProfile.GATT)
+        for (device in connectedDevices) {
+            this.gattServer.cancelConnection(device)
+        }
         this.gattServer.close()
```

Comment:
Shouldn't we close the gattServer when we stop advertising?

Reply:
I've moved the close to the `stopAdvertising` function.